pkgs/development/python-modules/notebook/default.nix

```diff
@@ -24,11 +24,11 @@
 
 buildPythonPackage rec {
   pname = "notebook";
-  version = "5.4.1";
+  version = "5.5.0";
```

Comment:
This breaks the build:

```
ERROR: Failure: ImportError (No module named selenium.webdriver.common.by)
```

Reply:
Adding `selenium` to `checkInputs` fixes it (selenium changelog: https://github.com/jupyter/notebook/blob/master/docs/source/changelog.rst).

Reply:
I don't think we should test selenium at all - those tests are too brittle in my experience, the hole test suite could be deleted before building: https://github.com/jupyter/notebook/tree/ebe01762eb3e93b2c3f525eceb27ca6dd9611000/notebook/tests/selenium